VoxelSniperCore/src/main/java/com/github/kevindagame/brush/polymorphic/PolyBrush.kt

```diff
@@ -104,7 +104,11 @@ class PolyBrush(
     private fun getPositions(v: SnipeData): MutableList<BaseLocation> {
         val positions = initPositions(v)
         val newPositions = mutableListOf<BaseLocation>()
-        val radiusSquared = (v.brushSize + if (smooth) SMOOTH_CIRCLE_VALUE else VOXEL_CIRCLE_VALUE).pow(2)
+        var radiusSquared = 0.0
+        try {
+            radiusSquared = (v.brushSize + if (smooth) SMOOTH_CIRCLE_VALUE else VOXEL_CIRCLE_VALUE).pow(2)
+        } catch (ignored: IllegalStateException) {
+        }
```

Comment:
When does this throw IllegalStateException?? If radiusSquared==0?

Reply:
If the brush isn't spherical, it doesn't have the smooth property

Reply:
Right now _radiusSquared_ would default to 0 when _smooth_ is not defined than? Don't think that is correct?
Also, could we check if _smooth_ exists in another way, that's more readable? Otherwise, please add a comment why the IllegalStateException might be fired, since this wasn't obvious to me in the code.